keras/utils/visualize_util.py

```diff
@@ -14,7 +14,8 @@
 
 
 def layer_typename(layer):
-    return type(layer).__module__ + "." + type(layer).__name__
+    # return type(layer).__module__ + "." + type(layer).__name__
```

Comment:
Oh, sorry. That got mixed in accidentally. I'll remove it tomorrow morning!
On Jan 4, 2016 20:10, "François Chollet" <notifications@github.com> wrote:

> In keras/utils/visualize_util.py
> <https://github.com/fchollet/keras/pull/1403#discussion_r48803181>:
>
> > @@ -14,7 +14,8 @@
> >
> >
> >  def layer_typename(layer):
> > -    return type(layer).__module__ + "." + type(layer).__name__
> > +    # return type(layer).__module__ + "." + type(layer).__name__
>
> What's up with this line?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fchollet/keras/pull/1403/files#r48803181>.
>